src/main/webapp/index.html

```diff
@@ -1,49 +1,106 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <meta charset="UTF-8">
-        <title>Search Page</title>
-        <link rel="stylesheet" href="search-style.css">
-        <script src="https://apis.google.com/js/platform.js" async defer></script>
-        <meta name="google-signin-client_id" content="12523748853-6h3fnh1kppht9s895ist2b5222ifh2u7.apps.googleusercontent.com">
-        <script src="search-script.js"></script>
-        <script src="login-script.js"></script>
-    </head>
-    <body>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Search Page</title>
+    <link rel="stylesheet" href="search-style.css" />
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.1/css/bootstrap.min.css"
+      integrity="sha384-VCmXjywReHh4PwowAiWNagnWcLhlEJLA5buUprzK8rxFgeH0kww/aWY76TfkUoSX"
+      crossorigin="anonymous"
+    />
+    <script src="https://apis.google.com/js/platform.js" async defer></script>
+    <meta
+      name="google-signin-client_id"
+      content="12523748853-6h3fnh1kppht9s895ist2b5222ifh2u7.apps.googleusercontent.com"
+    />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Arvo"
+    />
+  </head>
+  <body id="body" class="body">
+    <form id="form-signin">
+      <div class="container">
+        <h1 class="ml11">
+          <span id="t1">Course</span>
+          <span class="text-wrapper">
+            <span class="line line1"></span>
```

Comment:
If you were just reading through the css you'd have a hard time telling what "line" vs "line1" style.

Reply:
note sure how line1 got in there almost certain it is not needed